### PR TITLE
Fix minor references to auto-senescence in code

### PR DESCRIPTION
**Pull request by mdr0id**
Fix references

### CODE REVIEW

File: doc/release-notes/release-notes-1.0.13-rc1.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Jack Grigg (37):
       Fix NPE in rpc_wallet_tests
       make-release.py: Versioning changes for 1.0.13-rc1.
       make-release.py: Updated manpages for 1.0.13-rc1.
-      Change auto-senescence cycle to 16 weeks
+      Change end of support halt cycle to 16 weeks
```
Comment (daira) [inline doc/release-notes/release-notes-1.0.13-rc1.md]: "end-of-support", since it is a compound adjective.

**Comment by leto** [comment]
I am not sure it makes sense to change the content of older release notes, just my 2 zatoshis.

**Comment by bitcartel** [comment]
I agree with @leto, the older release notes should not be updated, for (1) historical purposes (2) they were auto-generated with a script.

**Comment by bitcartel** [comment]
Please squash all the commits into a single one, since this PR is now a one line change.

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**